grails-app/assets/javascripts/streama/controllers/dash-ctrl.js

```diff
@@ -23,12 +23,12 @@ angular.module('streama').controller('dashCtrl',
 
 		$scope.showDetails = function (media) {
 			//modalService.mediaDetailModal((media.tvShowId || media.id), media.mediaType); //{videoId: data.id}
-      modalService.mediaDetailModal({mediaId: (media.tvShowId || media.id), mediaType: media.mediaType});
+      modalService.mediaDetailModal({mediaId: (media.tvShowId || media.id), mediaType: media.mediaType, isApiMovie: false});
```

Comment:
you dont really need isApiMovie: false here, do you? because that is kind of the default..